depends/Makefile

```diff
@@ -33,6 +33,7 @@ WORK_PATH = $(BASEDIR)/work
 BASE_CACHE ?= $(BASEDIR)/built
 SDK_PATH ?= $(BASEDIR)/SDKs
 NO_QT ?=
+PROTOBUF ?=
```

Comment:
16871: should add/update `DEP_OPTS` in `matrix.sh` with `"PROTOBUF=1"` for builds with bip70 enabled (basically for every build besides `linux32`) cause otherwise all of them are build with no bip70 support in CI (see CI logs).